src/components/Tabs.js

```diff
@@ -22,13 +22,14 @@ const TabsFactory = ({ type = 'standard', useTabsInMobile, ...props }) => {
   if (!TabsComponent) throw new Error(`Unknown Tabs type: ${type}`);
 
   return (
-    <TabsComponent {...tabsProps} />
+    <TabsComponent ref={this.props.tabsRef} {...tabsProps} />
```

Comment:
I'm not sure how this should work on `Tabs`, can you remove it for now? I'll remove it from the issue todos as well.